themes/bootstrap3/templates/RecordDriver/Pazpar2/result-list.phtml

```diff
@@ -18,32 +18,32 @@
     <?=$thumbnail ?>
   <?php endif; ?>
   <div class="media-body">
-    <div>
+    <h2>
       <b>
         <?=$this->record($this->driver)->getTitleHtml()?>
       </b>
-    </div>
 
-    <div>
       <?php $summAuthors = $this->driver->getPrimaryAuthorsWithHighlighting(); ?>
       <?php if (!empty($summAuthors)): ?>
-        <?=$this->transEsc('by')?>
-        <?php $authorCount = count($summAuthors); ?>
-        <?php foreach ($summAuthors as $i => $summAuthor): ?>
-          <a href="<?=$this->record($this->driver)->getLink('author', $this->highlight($summAuthor, null, true, false))?>" class="result-author"><?=$this->highlight($summAuthor)?></a><?=$i + 1 < $authorCount ? ',' : ''?>
-        <?php endforeach; ?>
+        <br>
+        <span class="authors">
```

Comment:
Instead of a `<br>`, can we use a block element like `<div>` or block styles for authors?

Reply:
Thanks for the hint @crhallberg !
I removed the breaks before authors and set `display: block`.